### PR TITLE
[10.0][MIG] mrp_bom_structure_xlsx_level_1

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

Depends on:
- [x] (https://github.com/OCA/manufacture-reporting/pull/27)

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Tech and functional review

**Comment by JordiBForgeFlow** [comment]
@darshan-serpent can you review?

File: mrp_bom_structure_xlsx_level_1/README.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to 'Manufacturing / Products / Bill of Materials'
+#. Select a BOM or more BOMS
+#. Go to 'More / Export BOM Structure to Excel'.
```
Comment (serpentcs-dev1) [inline mrp_bom_structure_xlsx_level_1/README.rst]: Go to `'Action /  Export BOM Structure (Level 1) to Excel'`

**Review by serpentcs-dev1 — APPROVED** [review]
LGTM :+1:

**Pull request merged**